src/scripts/build.sh

```diff
@@ -66,6 +66,12 @@ if [ -n "$PARAM_CACHE_FROM" ]; then
   build_args+=("--cache-from=$PARAM_CACHE_FROM")
 fi
 
+if [ -n "$PARAM_CACHE_TO" ]; then
+  docker buildx create --name cache --use
+  docker buildx use cache
+  build_args+=("--cache-to=$PARAM_CACHE_TO" --load)
```

Comment:
Why is `--load` mandatory? I tested with a `Run with SSH` on CircleCI, and the load seems not necessary?